Sources/TuistCore/Models/LintingIssue.swift

```diff
@@ -8,48 +8,61 @@ public struct LintingError: FatalError, Equatable {
 }
 
 /// Linting issue.
-public struct LintingIssue: CustomStringConvertible, Equatable {
-    public enum Severity: String {
-        case warning
-        case error
-    }
-
-    // MARK: - Attributes
-
-    public let reason: String
-    public let severity: Severity
+public class LintingIssue: CustomStringConvertible, Equatable {
```

Comment:
@kwridan This was my initial approach , but I did not want the type of `[LintingIssue]` to be able to print warnings as it might lead to some mistakes where a linting error is not thrown before the generation begins. Any thoughts on this?

Reply:
I see the concern, in this case I think it's ok - the caller / client of the lint warnings can process them as they see fit :) - in this case `TuistKit.Generator` is responsible for orchestrating when lint errors are printed / thrown and when warning are displayed.